appveyor.yml

```diff
@@ -10,7 +10,7 @@ environment:
   bundler_url: https://rubygems.org/downloads/bundler-1.9.9.gem
 
   matrix:
-    - ruby_version: "193"
+    - ruby_version: "21"
```

Comment:
I actually just noticed this, but since 1.9.3 is so very obsolete and has long been unsupported, I don't think its worth changing winrm v2 to support it.